subsys/bluetooth/controller/ll_sw/ull_llcp_remote.c

```diff
@@ -490,6 +491,12 @@ static void rr_st_idle(struct ll_conn *conn, uint8_t evt, void *param)
 			const bool central = !!(conn->lll.role == BT_HCI_ROLE_CENTRAL);
 			const bool with_instant = proc_with_instant(ctx);
 
+			/*
+			 * Pause local incompatible procedure
+			 * in case we run a procedure with instant
+			 */
+			rr_set_collision(conn, with_instant);
+
```

Comment:
Going from memory, but I got procedure collissions, resulting in termination of connection when roughly doing the following:
remote initiates phy update
local initates phy update via HCI, and gets a REJECT PDU for this
IIRC remote queues another phy update request
local initiates a 2nd phy update via HCI

The 2nd phy update triggers a connection termination

Note that it is legal for the local procedure to initiate and send a first (REQ) PDU, but it will then receive a REJECT_IND if an incompatible procedure comes from central

Reply:
Going from memory, but I got procedure collissions, resulting in termination of connection when roughly doing the following:
remote initiates phy update
local initates phy update via HCI, and gets a REJECT PDU for this
IIRC remote queues another phy update request
local initiates a 2nd phy update via HCI

The 2nd phy update triggers a connection termination

Note that it is legal for the local procedure to initiate and send a first (REQ) PDU, but it will then receive a REJECT_IND if an incompatible procedure comes from central

Reply:
I will revert this, until I can get a unit-test in place to reproduce this behaviour